src/main/java/org/swisspush/redisques/RedisQues.java

```diff
@@ -1302,4 +1356,60 @@ private QueueConfiguration findQueueConfiguration(String queueName) {
         }
         return null;
     }
+
+    /**
+     * Retrieve the number of items of the requested queue
+     */
+    private void getQueueItemsCount(Message<JsonObject> event) {
+        String queue = event.body().getJsonObject(PAYLOAD).getString(QUEUENAME);
+        redisAPI.llen(queuesPrefix + queue, new GetQueueItemsCountHandler(event));
+    }
+
+    /**
+     * Retrieve the number of items of many queues
+     */
+    private void getQueuesItemsCount(Message<JsonObject> event) {
+        Result<Optional<Pattern>, String> filterPattern = MessageUtil.extractFilterPattern(event);
+        getQueuesItemsCount(event, filterPattern);
+    }
+
+    /**
+     * Retrieve the size of the queues matching the given filter pattern
+     */
+    private void getQueuesItemsCount(Message<JsonObject> event,
+        Result<Optional<Pattern>, String> filterPatternResult) {
+        if (filterPatternResult.isErr()) {
```

Comment:
```java
    private void getQueuesItemsCount(Message<JsonObject> event,
        Result<Optional<Pattern>, String> filterPatternResult) {
        if (filterPatternResult.isErr()) {
```

Misleading parameter indentation. See also my other comment where I listed a whole bunch of better alternatives.

Reply:
As before, will force non standard IntelliJ indentation.